views/add_event.hbs

```diff
@@ -1,76 +1,48 @@
 <div class="add-eventContainer">
-  <h3 class="userAddEvent">add event</h3>
+  <h3 class="userAddEvent">Add Event</h3>
 
   <form class="add-event-form" action="/add-event" method="post">
-    <label>Title: </label>
-    <input class="form-control" type="text" name="name" placeholder="title" required>
+    <label>Name </label>
+    <input class="form-control" type="text" name="name_english" placeholder="English" required>
+    <input class="form-control" type="text" name="name_arabic" placeholder="Arabic" required>
```

Comment:
Maybe just make the english input required for now - same with description. We'll improve this later when we add client-side validation :v: